runtime/parachains/src/inclusion.rs

```diff
@@ -962,7 +958,7 @@ impl<T: Config> CandidateCheckContext<T> {
 			Error::<T>::NotCollatorSigned,
 		);
 
-		let validation_code_hash = <paras::Pallet<T>>::validation_code_hash_at(para_id, now, None)
+		let validation_code_hash = <paras::Pallet<T>>::current_code_hash(para_id)
```

Comment:
i.e. this was previously future-proof changes to add asynchronous backing (old relay parent where code hadn't yet changed) but is not future-proof anymore

Reply:
Probably fine, from a product perspective - if we build parachains 1 block in advance then a code upgrade block would lead to a 6 second service interruption. Really not a problem.